app/models/katello/host/info_provider.rb

```diff
@@ -16,6 +16,15 @@ def host_info
         if host.content_facet.present?
           info['parameters']['kickstart_repository'] = host.content_facet.kickstart_repository.try(:label)
         end
+
+        if (rhsm_url = host.content_source&.rhsm_url)
+          host['parameters']['rhsm_url'] = rhsm_url.to_s
+        end
+
+        if (content_url = host.content_source&.pulp_content_url)
+          host['parameters']['content_url'] = content_url.to_s
```

Comment:
s/host/info ?

Reply:
I really suffer from not having a proper dev env running. My bad :(

Reply:
No worries! For awareness, forklift has a centosX-katello-devel-stable box that is built nightly and published to vagrant cloud. It spins up as fast as you can download the image!